draft/implementation-notes/index.html

```diff
@@ -602,7 +602,154 @@ <h2>Forward Delta</h2>
         </li>
       </ul>
     </section>
+    
+    <section>
+      <h2>Guidance on Updating OCFL Object Versions</h2>
+      <p>
+      The OCFL is designed to be a specification that covers objects at rest and consequently
+      does not specify in detail update and file locking mechanisms since these are implementation
+      dependent features. Nevertheless, this section includes some basic guidance on how to 
+      update OCFL objects in a manner that tries to ensure that updates are limited to a single  
+      transaction and that failures are detectable and recoverable.
```

Comment:
Do we need to point out that advice here will result in objects that cannot be validated during update, validation should be performed only when updates are complete?

Reply:
Noted.